Rasa_Bot/Makefile

```diff
@@ -0,0 +1,5 @@
+types:
+	pytype --keep-going actions tests
```

Comment:
Add this to the workflow/build. Just see how it works when we run it on everything.

Reply:
Add makefile to root.